### PR TITLE
ENH: Move to query package

### PR DESCRIPTION
**Pull request by gmatthews20**
### Description:

Query Library has been moved to its own dedicated repository: https://github.com/stfc/openstack-query-library

This PR removes the query library directories and changes the stackstorm workflows to use the new package from the query library repository

### Special Notes:

---

### Submitter:

Have you (where applicable):

* [x] Added unit tests?
* [x] Checked the latest commit runs on Dev?
* [x] Updated the example config file(s) and README?

---

### Reviewer

Does this PR:

* [x] Place non-StackStorm code into the `lib` directory?
* [x] Have unit tests for the action/sensor and `lib` layers?
* [x] Have clear and obvious action parameter names and descriptions?

### CODE REVIEW

File: docs/DEVELOPER_NOTES.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ There are currently 4 API submodules:
 
 4. Openstack Query API - A python library developed by the Cloud Team to make querying Openstack easier.
    - This is in active development, and we're aiming to move this out of this pack to be a standalone library.
```
Comment (anish-mudaraddi) [inline docs/DEVELOPER_NOTES.md:56]: you can remove this line

File: lib/workflows/send_errored_vm_email.py [hunk 2]
```diff
@@ -25,13 +23,11 @@ def find_servers_with_errored_vms(
     server_query = ServerQuery()
     if time_variable > 0:
         server_query.where(
-            QueryPresetsDateTime.OLDER_THAN,
-            ServerProperties.SERVER_LAST_UPDATED_DATE,
+            "olser_than",
```
Comment (Dmitry-Popovichev) [inline lib/workflows/send_errored_vm_email.py:26]: Should this be `older_than`?

File: tests/lib/openstack_api/test_openstack_hypervisor.py [hunk 3]
```diff
@@ -6,119 +7,119 @@ def test_get_state_running():
...
+    assert state == HypervisorState.DRAINED
 
 
 def test_get_state_unkown_status():
     """
     Test hypervisor state is unknown when missing status
     """
     hypervisor = {
-        "hypervisor_uptime": "13:37:30 up 100 days,  4:29, 19 users,  load average: 0.04, 0.04, 0.0",
+        "hypervisor_uptime_days": 100,
```
Comment (Dmitry-Popovichev) [inline tests/lib/openstack_api/test_openstack_hypervisor.py:66]: Is there an upper and/or lower limit to this or can it be any amount of days?

File: lib/workflows/send_errored_vm_email.py (hunk 2 above)
Comment (gmatthews20) [reply lib/workflows/send_errored_vm_email.py]: Yep

File: docs/DEVELOPER_NOTES.md (hunk 1 above)
Comment (gmatthews20) [reply docs/DEVELOPER_NOTES.md]: Done

File: tests/lib/openstack_api/test_openstack_hypervisor.py (hunk 3 above)
Comment (Dmitry-Popovichev) [reply tests/lib/openstack_api/test_openstack_hypervisor.py:66]: Cool cool, was just wondering if there was a test case for 0 or an upper limit if it existed.

**Review by Dmitry-Popovichev — APPROVED** [review]

**Pull request merged**